portfolio/src/main/java/com/google/sps/servlets/ListOfFacts.java

```diff
@@ -0,0 +1,32 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+@WebServlet("/facts")
+public class ListOfFacts extends HttpServlet { 
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        ArrayList<String> array = new ArrayList<String>(Arrays.asList("I used to play chess in highschool.", "I play soccer since I am 4 years old.", "I'm always smiling."));
```

Comment:
It seems like this array could get built once for the class and then reused in each call to doGet rather than rebuilding the array every time.